libs/core/src/lib/feed-list-item/directives/line-clamp.directive.ts

```diff
@@ -0,0 +1,207 @@
+import { coerceNumberProperty } from '@angular/cdk/coercion';
+import {
+  Directive,
+  ElementRef,
+  EventEmitter,
+  Input,
+  AfterViewInit,
+  OnDestroy,
+  OnChanges,
+  Output,
+  Renderer2
+} from '@angular/core';
+import { fromEvent, Subscription } from 'rxjs';
+import { debounceTime, distinctUntilChanged } from 'rxjs/operators';
+
+@Directive({
+  selector: '[fdLineClampTarget]',
```

Comment:
We should create both `kebab-case` and `lowerCamelCase` directive selectors (for some reason).

Reply:
Ok, updated